tab_menu.py

```diff
@@ -110,8 +110,9 @@
         "caption": "Copy File Path",
         "children":
         [
+            { "command": "tabs_extra_file_path", "args": {"group": -1, "index": -1, "path_type": "name"}, "caption": "File Name" },
             { "command": "tabs_extra_file_path", "args": {"group": -1, "index": -1, "path_type": "path"}, "caption": "Full Path" },
-            { "command": "tabs_extra_file_path", "args": {"group": -1, "index": -1, "path_type": "name"}, "caption": "File Name" }
+            { "command": "tabs_extra_file_path", "args": {"group": -1, "index": -1, "path_type": "path_uri"}, "caption": "Full Path as URI" },
```

Comment:
Can we just call this `File URI`?  A URI implies it is going to be the full path.